### ISSUE TITLE
General performance problem

### ISSUE THREAD

**Issue by Gert-69115** [abap2xlsx/abap2xlsx#448]
We noticed that putting the lines of a huge internal table to an Excel worksheet cell by cell can be very time-consuming (up to hours). 
Are we doing something wrong here or is it a known issue?
Regards,
  Gert

**Comment by gregorwolf** [comment]
Hi Gert,

have you tried using bind_table as shown in the demo report ZDEMO_EXCEL3? Can you specify the "huge"? How many lines, columns?

Best regards
Gregor

**Comment by ivanfemia** [comment]
Hi Gent,

have you tried the huge file writer?
It has some limitation on the formatting but it works for big files.

Ivan

**Comment by Gert-69115** [comment]
Hello Ivan & Gregor.
Thank you for your quick reply.

Sometimes the internal tables we work with have many entries, e.g. the filling of a table with 40.000 lines and 20+ columns takes a bit more than two minutes. We do not download the created data, but take the generated binary string and store it in the KPro. So it's not a problem with the physical download.

The BIND_TABLE did not help here, since it does almost the same as our procedure in which fill the cells individually.
Then I changed the 'writer' from 'writer_2007' to 'writer_hugefile'. This reduced the time needed for the generation of the Excel data significantly, at least in the first tests I ran.

Thank you again for your assistance!

Gert
